### ISSUE TITLE
`exp show`: Rename / Split to separate command the current `--html`

### ISSUE THREAD

**Issue by daavoo** [iterative/dvc#7165]
> My biggest question is whether this belongs in `dvc exp show` or might be better as a separate command, like `dvc exp plot`? A `dvc exp plot` command could also potentially show other things, like `dvc plots diff` between experiments, or checkpoint trends (like dvclive plots).

_Originally posted by @dberenbaum in https://github.com/iterative/dvc/issues/6933#issuecomment-973159973_

> Why something as generic and conceptually disconnected as "HTML"? How about `--parallel-coordinates-plot`/`--pcp`? Or a separate command as @dberenbaum mentioned

_Originally posted by @jorgeorpinel in https://github.com/iterative/dvc/issues/6933#issuecomment-996059215_

> To be honest `--html` for me would mean an HTML table, not plot. I would consider renaming this to be explicit about its meaning. I think `--plot` is a bit more explicit. But can limit us later if we decide to introduce more types of plots. Probably `--pcp` is the best unfortunately.

_Originally posted by @shcheklein in https://github.com/iterative/dvc/issues/6933#issuecomment-996459692_

**Comment by daavoo** [comment]
I think the best option would be to move to a separate command `dvc exp plot` with no flags for now. If more type of plots are added, we can introduce `--pcp` flags for whatever new type of plot.

It also opens possibility for further integration of `plots` and `experiments`

**Comment by jorgeorpinel** [comment]
Yeah I can see why it was included to `exp show` initially, as it's basically the same info. But the presentation is completely different plus indeed maybe we want to add other plots later so I'd vote for a separate command.

**Comment by daavoo** [comment]
> Yeah I can see why it was included to `exp show` initially, as it's basically the same info. But the presentation is completely different plus indeed maybe we want to add other plots later so I'd vote for a separate command.

One worry that I have is that adding a separate command, no matter the name, might create more doubts from the user's perspective.

If we name `dvc exp plot`/`dvc exp report` with the current features, users might ask where are the other plots (i.e. live plots) or why it's named report if it's only a PCP plot.

Proper `exp plot`/`exp report` requires mid-term engineering and product work in order to be implemented properly but the issue with the `html` name needs to be fixed short-term. 

I'm wondering if just rename `--html` to `--pcp` for now until a separate command is consolidated 😅

**Comment by jorgeorpinel** [comment]
`--pcp` is the option with the most implicit votes already and definitely an improvement so I believe we can go ahead with that for now indeed.

I don't think users would be confused about `exp plot` only having one plot though.

BTW where are live plots documented?

**Comment by daavoo** [comment]
Extracted the new subcommand discussion to  #7191

**Comment by daavoo** [comment]
> I don't think users would be confused about `exp plot` only having one plot though.

I meant that there could be confusion with `dvc plots` because experiments can also generate that kind of plot but they won't be shown (with current features) in `dvc exp plot`.
 
> BTW where are live plots documented?

They are not documented with that name anywhere. I was referring to https://dvc.org/doc/dvclive/dvclive-with-dvc#html-report (I think I used it because I recently heard the term `live plots` in VSCode demo or something)

**Comment by jorgeorpinel** [comment]
> experiments can also generate that kind of plot but they won't be shown in dvc exp plot

I.e. if your experiment code implements outputting a PCP? I see. Edge case though, I think!

Thanks for clarifying on live plots. That's a feature of DVCLive so I wouldn't expect it to be in a DVC command -- esp. since DVCLive works standalone.

**Comment by skshetry** [comment]
I'd argue that since we are not sure about `--pcp` either, status quo should remain. We should put efforts on discussions in #7191. `--pcp` is not obvious either, and unknown abbreviation at that.

**Comment by daavoo** [comment]
> I'd argue that since we are not sure about --pcp either, status quo should remain. We should put efforts on discussions in #7191. --pcp is not obvious either, and unknown abbreviation at that.

It's a pity that we have already released `--html` but it's very recent and I wouldn't say there is status quo yet; especially since docs P.R. is still open.

I don't think that `--pcp` is the final solution and agree on focusing on #7191 . However, I do think `--pcp` it's better than `--html`. The abbreviation is not common but we can teach on docs and at least it's not ambiguous, unlike `--html`.
